### ISSUE TITLE
When selecting a date with “submitted on” or Expires on” the other dates disappear, can this not happen

### ISSUE THREAD

**Issue by ger630** [sweepsatlas/issues#54]

**Comment by sixty4bit** [comment]
It can, but should it? I think you are the only person that would use that capability.

**Comment by ger630** [comment]
Yes I think it might be useful for people....you can poll the mods if you want.

**Comment by sixty4bit** [comment]
Poll created in the UX group.

**Comment by sixty4bit** [comment]
Shocker, this one split basically down the lines of power user vs normie. The normies picking one day at a time. Humorously, Pippismommy is the tie breaker. I'm going to leave it as is for now, but I will think about how to cleanly solve this issue because you super users are important to me.
